src/app/development/middleware.js

```diff
@@ -1,3 +1,4 @@
+/* istanbul ignore file */
```

Comment:
Is this for sonar? could we add it to the exclusions in `sonar-project.properties` instead